base_station/gui/src/components/AutonTask.vue

```diff
@@ -44,14 +44,16 @@
     <div class="box waypoints light-bg">
       <AutonWaypointEditor v-bind:odom="odom" v-bind:AutonDriveControl="AutonDriveControl"/>
     </div>
-    <div class="box angles light-bg">
+    <div class="box cameras light-bg">
+      <AutonCameras v-if="!this.autonEnabled" />
```

Comment:
Did you test this functionality with the big auton enabled button in the top right of the GUI?

Reply:
yeah, works as intended

Reply:
coolio